activation/atxdb.go

```diff
@@ -124,12 +124,26 @@ func (db *ActivationDb) CalcActiveSetFromView(a *types.ActivationTx) (uint32, er
 	if err != nil {
 		return 0, err
 	}
-	activesetCache.Add(common.BytesToHash(viewBytes), counter)
+	activesetCache.Add(viewHash, counter)
 
 	return counter, nil
 
 }
 
+func calcSortedViewHash(atx *types.ActivationTx) ([32]byte, error) {
```

Comment:
add a small UT, to make sue this doesnt brake in the future

Reply:
done